content/tbb/tbb-25/contents.lr

```diff
@@ -5,5 +5,18 @@ title: I'm having a problem with NoScript.
 seo_slug: problem-noscript
 ---
 description: 
-Please see the <mark><a href="https://noscript.net/faq">NoScript FAQ</a></mark>.
-If you believe this is a Tor Browser issue, please report it on our <mark><a href="https://trac.torproject.org/">bug tracker</a></mark>.
+NoScript is a web browser extension.
+It can block or unblock active web content such as JavaScript, Flash and other plugins. 
+By default, several plugins, including Flash, are blocked by NoScript. 
+NoScript provides protection against Cross-site scripting (XSS), CSRF (Cross-site request forgery), clickjacking, man-in-the-middle attacks, and DNS rebinding.
+
+You can enable or disable NoScript on Tor Browser by going to the add-ons manager. 
```

Comment:
We should not teach users how to disable NoScript like this. They can set it to the "standard" level in Security Settings.